<ui version="4.0" >
 <class>LoginSettingsBase</class>
 <widget class="QWidget" name="LoginSettingsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>LoginSettingsWidget</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="1" >
    <widget class="KLineEdit" name="kcfg_GmailUsername" />
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="usernameLabel" >
     <property name="text" >
      <string>Username:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>usernameTxt</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="passwordLabel" >
     <property name="text" >
      <string>Password:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>passwordTxt</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QCheckBox" name="kcfg_PasswordFromWallet" >
     <property name="text" >
      <string>&amp;Use KDE Wallet to store password</string>
     </property>
     <property name="shortcut" >
      <string>Alt+U</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer name="spacer1" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="KLineEdit" name="gmailPassword" >
     <property name="passwordMode" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_GmailUsername</tabstop>
  <tabstop>gmailPassword</tabstop>
  <tabstop>kcfg_PasswordFromWallet</tabstop>
 </tabstops>
 <includes>
  <include location="local" >klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
